frontend/javascripts/libs/zipjs_wrapper.ts

```diff
@@ -0,0 +1,14 @@
+class TransFormStream {}
+
+let mockedWindow = false;
```

Comment:
Omitting it doesn't seem to break anything, so I left it out.
Just for clarification. Technically this *could* have been used somewhere else because the scope of the variable is global, right? (https://www.w3schools.com/js/js_scope.asp)

Reply:
> Technically this could have been used somewhere else because the scope of the variable is global, right? (https://www.w3schools.com/js/js_scope.asp)

The linked article is a bit confusing (generally, w3schools ist not the best resource; I recommend MDN instead). However, the specific [MDN article](https://developer.mozilla.org/en-US/docs/Glossary/Scope?retiredLocale=de) is also not perfect (it mentions module scope, but doesn't really explain when it comes into play).
To answer your question: no, it could not have been used somewhere else because the variable is module-scoped. That scoping is achieved during the compilation/bundling of all the module files. To get a truly global variable, one has to extend `window` (in a browser context) or `global` (in a node context).